src/prefect/blocks/notifications.py

```diff
@@ -80,6 +81,26 @@ class AppriseNotificationBlock(AbstractAppriseNotificationBlock, ABC):
         description="Incoming webhook URL used to send notifications.",
         examples=["https://hooks.example.com/XXX"],
     )
+    allow_private_urls: bool = Field(
+        default=True,
+        description="Whether to allow notifications to private URLs. Defaults to True.",
+    )
```

Comment:
This makes a ton of sense as a block-level setting.  This should work well with Prefect Cloud because Cloud will just ignore that there's a field `allow_private_urls` at all for any server-side block instantiations, but should still _return_ the value so that client-side instantiations have it.  Plus, with a default of `True`, all existing blocks will continue to work the same.  Am I thinking about that right?

Reply:
Yup, that articulates it well!